spec/models/like_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  user = User.new(name: 'najib', posts_counter: 0)
+  post = Post.new(title: 'man', comments_counter: 0, likes_counter: 0)
+  it 'is vlaid with a user and a post' do
+    like = Like.new(user:, post:)
+    expect(like).to be_valid
+  end
+
+  it 'is invalid without user' do
+    lik = Like.new(post:)
+    expect(lik).to be_invalid
+  end
+
+  it 'the like sould not be present' do
+    Like.new(user:, post:)
+    fountp = Post.find_by(id: post.id)
+    expect(fountp).to_not be_present
+  end
+  # method test
+  it 'like counter method' do
+    user = User.create(name: 'najib', posts_counter: 0)
+    post = Post.create(title: 'man', comments_counter: 0, likes_counter: 0)
+    Comment.create(user:, post:)
+    expect(post.likes_counter).to eq('0')
+  end
+  it 'updates the posts likes_counter' do
+    user = User.create(name: 'najib', posts_counter: 0)
+    post = Post.create(title: 'man', comments_counter: 0, likes_counter: 0)
+    like = Like.create(user:, post:)
+    expect { like.update_likes_counter }.to_not(change { post.likes_counter })
+  end
+end
```

Comment:
- [ ] Great job so far @najibullahjafari I see that you have included a test for the `update_likes_counter` method. However, your test doesn't seem to be correct and doesn't meet the previous reviewer's suggestion. As the previous reviewer mentioned,  the test should check whether creating a new like and calling the `update_likes_counter` method increments the `likes_counter` by 1. So you can refer to the previous reviewers [example test ](https://github.com/najibullahjafari/ruby_social_media/pull/3/files#r1349506645)above and test that method properly 😉